build.gradle.kts

```diff
@@ -3,13 +3,14 @@ plugins {
 }
 
 group = "com.playmonumenta.papermixins"
-version = "1.0.2"
+version = "1.0.4"
```

Comment:
shouldn't this version bump be to 1.0.3? or is that because of the other PR

Reply:
1.0.3 was already published when testing (should've been 1.0.3-snapshot but whatever), and I can't be bother to figure out gradle caching.